run_tests.sh

```diff
@@ -22,66 +14,39 @@ set -ex
 # golangci-lint (github.com/golangci/golangci-lint) is used to run each each
 # static checker.
 
-# To run on docker on windows, symlink /mnt/c to /c and then execute the script
-# from the repo path under /c.  See:
-# https://github.com/Microsoft/BashOnWindows/issues/1854
-# for more details.
-
-# Default GOVERSION
-[[ ! "$GOVERSION" ]] && GOVERSION=1.12
 REPO=dcrd
-
-testrepo () {
-  GO=go
-
-  $GO version
-
-  # binary needed for RPC tests
-  env CC=gcc $GO build
-  cp "$REPO" "$GOPATH/bin/"
-
-  # run tests on all modules
-  ROOTPATH=$($GO list -m)
-  ROOTPATHPATTERN=$(echo $ROOTPATH | sed 's/\\/\\\\/g' | sed 's/\//\\\//g')
-  MODPATHS=$($GO list -m all | grep "^$ROOTPATHPATTERN" | cut -d' ' -f1)
-  for module in $MODPATHS; do
-    echo "==> ${module}"
-    env CC=gcc $GO test -short -tags rpctest ${module}/...
-
-    # check linters
-    MODNAME=$(echo $module | sed -E -e "s/^$ROOTPATHPATTERN//" \
-      -e 's,^/,,' -e 's,/v[0-9]+$,,')
-    if [ -z "$MODNAME" ]; then
-      MODNAME=.
-    fi
-    (cd $MODNAME && \
-      golangci-lint run --build-tags=rpctest --disable-all --deadline=10m \
-        --enable=gofmt \
-        --enable=gosimple \
-        --enable=unconvert \
-        --enable=ineffassign \
-        --enable=govet \
-        --enable=misspell \
-    )
-  done
-
-  echo "------------------------------------------"
-  echo "Tests completed successfully!"
-}
-
-DOCKER=
-[[ "$1" == "docker" || "$1" == "podman" ]] && DOCKER=$1
-if [ ! "$DOCKER" ]; then
-    testrepo
-    exit
-fi
-
-# use Travis cache with docker
-DOCKER_IMAGE_TAG=decred-golang-builder-$GOVERSION
-$DOCKER pull decred/$DOCKER_IMAGE_TAG
-
-$DOCKER run --rm -it -v $(pwd):/src:Z decred/$DOCKER_IMAGE_TAG /bin/bash -c "\
-  rsync -ra --filter=':- .gitignore'  \
-  /src/ /go/src/github.com/decred/$REPO/ && \
-  cd github.com/decred/$REPO/ && \
-  env GOVERSION=$GOVERSION GO111MODULE=on bash run_tests.sh"
+GO=go
```

Comment:
No need to keep this anymore since it was only used to be able to override with vgo.  I'd suggest nuking it and replacing all the refs below accordingly.

Reply:
done